File: co.edu.uniandes.fixml.toC++/test/test6.xml/Amt.cpp
#include <iostream> 

using namespace std;

class Amt {

	private:
  	string Amt_;
  	string Typ_;
 	
 	public: 

  	// default constructor
	Amt () {
		Amt_ = "0.00";
		Typ_ = "FMTM";
  	
  	}
  
  	// by parameters
  	Amt (		string Amt_p, string Typ_p){
	  
		Amt_ = Amt_p;
		Typ_ = Typ_p;
	  
  	}
};

File: co.edu.uniandes.fixml.toCSharp/test/test5.xml/Amt.cs


// Class for tag Amt
class Amt {

	string _Amt;
	string _Typ;
	
	
	// default constructor
	
	public Amt ( ) {   
		this._Amt = "0.00";
		this._Typ = "FMTM";
		
		
	}
	
	public Amt ( string Amt, string Typ ) {
		this._Amt = Amt;
		this._Typ = Typ;
		
	}
	
	
} 